python/rpdk/java/templates/generate/HandlerWrapper.java

```diff
@@ -108,8 +115,35 @@ public void testEntrypoint(
             response = ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.InternalFailure);
         } finally {
             writeResponse(outputStream, response);
+            outputStream.close();
+        }
+    }
+    {% else %}
+    public static void main(String[] args) throws IOException {
+        if (args.length != 1){
```

Comment:
one arg is just the stringified handler request?

Reply:
Yes